byway/src/main/java/com/google/sps/servlets/DestinationsServlet.java

```diff
@@ -0,0 +1,72 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.appengine.api.datastore.Key;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.ArrayList;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+
+/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+@WebServlet("/api/destinations")
+public class DestinationsServlet extends HttpServlet {
+
+    private final static class UserLocations{ 
```

Comment:
This may be worth moving to a separate file and having a stand alone class rather than a nested class. Especially if you plan on using this class in more ways in the future.

Reply:
done!